File: scripts/tutorial5/python/plotTaskFailureReason.py
from plotGenericLine import plot_generic_line

if __name__ == '__main__':
    print("--- Generating: Task Failure Reason Plots ---")

    plot_generic_line(1, 10, 'Failed Task due to VM Capacity (%)', 'ALL_APPS', 'percentage_of_failed', 'upper left')
    plot_generic_line(1, 11, 'Failed Task due to Mobility (%)', 'ALL_APPS', 'percentage_of_failed', 'upper left')
    plot_generic_line(5, 5, 'Failed Tasks due to WLAN (%)', 'ALL_APPS', 'percentage_of_failed', 'upper left')

File: scripts/scenario5/matlab/plotTaskFailureReason.m
function [] = plotTaskFailureReason()

    plotGenericLine(1, 10, 'Failed Task due to VM Capacity (%)', 'ALL_APPS', 'percentage_of_failed', 'NorthWest');
    
    plotGenericLine(1, 11, 'Failed Task due to Mobility (%)', 'ALL_APPS', 'percentage_of_failed', 'NorthWest');
    
    plotGenericLine(5, 5, 'Failed Tasks due to WLAN (%)', 'ALL_APPS', 'percentage_of_failed', 'NorthWest')

end